src/components/modal/social/modal.scss

```diff
@@ -0,0 +1,132 @@
+@import "../../../colors";
+@import "../../../frameless";
+
+.mod-social {
+    min-height: 15rem;
+    max-height: calc(100% - 8rem);
+    overflow: hidden;
+}
+
+.social-modal-header {
+    border-radius: 1rem 1rem 0 0;
+    box-shadow: inset 0 -1px 0 0 $ui-blue-dark;
+    background-color: $ui-blue;
+}
+
+.social-modal-content {
+    box-shadow: none;
+    width: 92%;
+    height: calc(100% - 0rem);
+    margin: 1rem auto 1.625rem;
+    font-size: .9375rem;
+}
+
+.social-row {
+    width: 100%;
+    margin-top: .5rem;
+    margin-bottom: .5rem;
+    align-items: start;
+}
+
+.social-spaced-row {
+    // width: 100%;
+    justify-content: space-between;
+    align-items: flex-end;
+}
+
+.social-row-right {
+    margin-left: auto;
+}
+
+.social-label-row {
+    width: 100%;
+    font-weight: bold;
+    margin-bottom: .5rem;
+    justify-content: space-between;
+    align-items: flex-end;
+}
+
+.social-label-title {
+    font-size: 1rem;
+    margin-right: 1.5rem;
+}
+
+.social-label-item {
+    margin-left: 1.5rem;
+    margin-right: .25rem;
+}
+
+.social-label-result {
+    color: $type-gray-75percent;
+    transition: opacity 100ms linear;
+}
+
+.link-section {
+    margin-top: .5rem;
+}
+
+.embed-section {
+    margin-top: 1rem;
+}
+
+.social-social-icon {
+    width: 2.75rem;
+    height: 2.75rem;
+    margin-right: .75rem;
+    background-repeat: no-repeat;
+    background-size: contain;
+}
+
+.social-twitter-icon {
+    background-image: url("/images/social/twitter.png");
+}
+
+.social-facebook-icon {
+    background-image: url("/images/social/facebook.png");
+}
+
+.social-google-classroom-icon {
+    background-image: url("/images/social/google-classroom.png");
+}
+
+.social-wechat-icon {
+    background-image: url("/images/social/wechat.png");
+}
+
+.social-form {
```

Comment:
much of this is copied from inplace-input.scss, but it wasn't clear to me if there was a way that made sense to refactor it so the code could be shared.